pkg/bundle/create/create.go

```diff
@@ -28,19 +28,19 @@ func CreateFull(rootDir string) error {
 		return err
 	}
 	logrus.Info(config)
-	if &config.Mirror.Ocp != nil {
+	if config.Mirror.Ocp.Channels != nil {
```

Comment:
This slice may still be empty if non-nil.
```suggestion
	if len(config.Mirror.Ocp.Channels) != 0 {
```